dspace-spring-rest/src/main/java/org/dspace/app/rest/converter/SubmissionFormConverter.java

```diff
@@ -142,10 +146,24 @@ private SubmissionFormFieldRest getField(DCInput dcinput, String formName) {
             }
         }
         inputField.setInput(inputRest);
-        inputField.setSelectableMetadata(selectableMetadata);
+        if (dcinput.isMetadataField()) {
+            inputField.setSelectableMetadata(selectableMetadata);
+        }
+        if (dcinput.isRelationshipField()) {
+            handleSelectableRelationships(dcinput, selectableRelationships);
+            inputField.setSelectableRelationships(selectableRelationships);
+        }
         return inputField;
     }
 
+    private void handleSelectableRelationships(DCInput dcinput, List<SelectableRelationship> selectableRelationships) {
```

Comment:
While this is a private method, I'd like to see some Javadocs or inline comments to describe what is going on here...it's not entirely straightforward

Reply:
The JavaDoc has been included, but the method has also been refactored since only one SelectableRelationship can be defined per metadata field.
This was a copy/paste error in the rest contract, and has also been adjusted in https://github.com/DSpace/Rest7Contract/pull/69